src/plugins/vis_types/pie/public/plugin.ts

```diff
@@ -45,24 +42,10 @@ export interface VisTypePieDependencies {
 export class VisTypePiePlugin {
   setup(
     core: CoreSetup<VisTypePiePluginStartDependencies>,
-    { expressions, visualizations, charts, usageCollection }: VisTypePieSetupDependencies
```

Comment:
We can also remove the expressions from the `VisTypePieSetupDependencies`

Reply:
My bad, sorry) Done.